app/src/main/res/layout/fragment_scheduler.xml

```diff
@@ -8,189 +8,272 @@
-                    layout="@layout/item_top_sheet"
-                    bind:description="@{@string/title_select_date}"
-                    bind:tabFilter="@{TabFilter.TAB_SELECT_DATE}"
-                    bind:topSheetLayout="@{layoutTopSheet}"
-                    bind:viewModel="@{viewModel}"
-                    />
-
-            </LinearLayout>
-
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="?attr/actionBarSize"
-                android:background="@color/colorPrimary"
-                android:gravity="center_vertical"
-                android:orientation="horizontal"
-                >
+                <android.support.v7.widget.RecyclerView
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="@color/color_grey_100"
+                    bind:layoutManager="@{viewModel.layoutManager}"
+                    bind:recyclerAdapter="@{viewModel.adapter}"/>
 
-                <TextView
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginLeft="@dimen/dp_20"
-                    android:layout_marginStart="@dimen/dp_20"
-                    android:layout_weight="1"
-                    android:text="@{viewModel.titleTopSheet}"
-                    android:textColor="@color/color_white"
-                    android:textSize="@dimen/sp_20"
-                    android:textStyle="bold"/>
-
-                <Spinner
+                <ProgressBar
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginEnd="@dimen/dp_20"
-                    android:layout_marginRight="@dimen/dp_20"
-                    bind:array="@{@stringArray/title_status}"
-                    bind:listener="@{viewModel.spinnerSelectListener}"/>
+                    android:layout_gravity="bottom|center"
+                    android:layout_marginBottom="@dimen/dp_16"
+                    android:visibility="@{viewModel.loadingMore?View.VISIBLE:View.GONE}"/>
 
-                <ImageView
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginEnd="@dimen/dp_20"
-                    android:layout_marginRight="@dimen/dp_20"
-                    android:onClick="@{()->viewModel.onClickTopSheet(layoutTopSheet)}"
-                    app:srcCompat="@drawable/ic_swipe_down_24dp"/>
-            </LinearLayout>
-        </LinearLayout>
+                <com.github.clans.fab.FloatingActionMenu
+                    android:id="@+id/menu_yellow"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="bottom|left"
```

Comment:
WARNING: [RtlHardcoded] Use "`start`" instead of "`left`" to ensure correct behavior in right-to-left locales
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)